src/execution/executor/ddl/alter_table.rs

```diff
@@ -0,0 +1,60 @@
+use crate::execution::executor::BoxedExecutor;
+use crate::types::tuple::Tuple;
+use crate::types::value::DataValue;
+use crate::{execution::ExecutorError, types::tuple_builder::TupleBuilder};
+use futures_async_stream::try_stream;
+use std::cell::RefCell;
+use std::sync::Arc;
+
+use crate::{
+    execution::executor::Executor, planner::operator::alter_table::AddColumnOperator,
+    storage::Transaction,
+};
+
+pub struct AddColumn {
```

Comment:
For alert functions that contain multiple operators, you can change the file name to add_column and wrap it in the alert folder.